### PR TITLE
Issue 136 - Added new action to list the PCM metrics

### PR DESCRIPTION
**Pull request by SreenidhiS1**

### CODE REVIEW

File: plugins/modules/power_system.py [hunk 1]
```diff
@@ -539,7 +587,7 @@ def run_module():
         mem_mirroring_mode=dict(type='str', choices=['none', 'sys_firmware_only']),
         metrics=dict(type='list', elements='str', choices=['LTM', 'STM', 'AM', 'CLTM', 'EM']),
         pend_mem_region_size=dict(type='str', choices=['auto', '16', '32', '64', '128', '256']),
-        action=dict(type='str', choices=['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm']),
+        action=dict(type='str', choices=['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm', 'list_pcm']),
```
Comment (naveenkandakur) [inline plugins/modules/power_system.py]: use the pcm_facts state

File: plugins/modules/power_system.py [hunk 2]
```diff
@@ -110,8 +110,9 @@
             - C(modify_hwres) Makes hardware resource configurations of specified I(system_name).
             - C(enable_pcm) Enables the Performance and Capacity Monitoring for specified types of utilization data.
             - C(disable_pcm) Disables the Performance and Capacity Monitoring for specified types of utilization data.
+            - C(list_pcm) Provides the status of the Performance and Capacity Monitoring of the managed system
```
Comment (naveenkandakur) [inline plugins/modules/power_system.py]: please provide more details of what details it shows

**Review by naveenkandakur — CHANGES_REQUESTED** [review]
add unit tests

File: plugins/modules/power_system.py (hunk 1 above)
Comment (SreenidhiS1) [reply plugins/modules/power_system.py]: done

File: plugins/modules/power_system.py [hunk 3]
```diff
@@ -110,8 +110,9 @@
...
         type: str
-        choices: ['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm']
+        choices: ['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm', 'list_pcm']
```
Comment (naveenkandakur) [inline plugins/modules/power_system.py]: this documentation still shows as list_pcm, which should be changed to ppm_facts

**Review by naveenkandakur — CHANGES_REQUESTED** [review]

File: plugins/modules/power_system.py (hunk 3 above)
Comment (SreenidhiS1) [reply plugins/modules/power_system.py]: done

File: plugins/modules/power_system.py (hunk 2 above)
Comment (SreenidhiS1) [reply plugins/modules/power_system.py]: Done

**Review by naveenkandakur — APPROVED** [review]

**Pull request merged**